tool/menu.lua

```diff
@@ -148,36 +148,55 @@ end
 
 local function randomizermenu()
   local selected = ui.select({
+    "New Seed on New Game    " .. randomizer.newgamenewseedui[randomizer.newgamenewseed],
     "Proficiency: " .. randomizer.proficiency,
```

Comment:
I'd argue that the proficiency is the more prominent setting and should be the topmost setting. NSonNG should be second IMO. Unrelated: It might also make sense to rename `proficiency` to `difficulty` everywhere, including the variables. What do you think?

Reply:
I had it second in earlier commits, but I put it first afterwards since it shows the menu sorted from longest to shortest.
I'll put an image comparison on discord so you can tell me what you think (alternate image links: [here](https://cdn.discordapp.com/attachments/272821008644308992/747728613826101298/unknown.png) and [here](https://cdn.discordapp.com/attachments/272821008644308992/747728680293367838/unknown.png).)

Reply:
I also prefer `difficulty` to `proficiency`, so I'll rename it.